bin/docker_build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Get the directory to build argument
+if [ -n "$1" ] && [ "$1" != "--rebuild" ]; then
+  directory_to_build=$1
+elif [ -n "$2" ] && [ "$2" != "--rebuild" ]; then
+  directory_to_build=$2
+fi
+
+# Build the Docker image, if the --rebuild flag was used or if an image hasn't been built yet
+
+if [ -z "$(docker images | grep wca-pdf-builder)" ]; then
+  image_exists=true
+  docker rmi wca-pdf-builder
+fi
+
+if [ "$1" = "--rebuild" ] || [ "$2" = "--rebuild" ] || [ "$image_exists" = true ]; then
+  docker build --tag wca-pdf-builder --build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g) .
+
+  # Make sure the command completed successfully
+  if [ $? != 0 ]; then
+    exit
+  fi
```

Comment:
This is what the `||` control flow operator in Bash is made for `try_some_command --arg "foo" || echo 'the command failed!'`. In this context, you probably want `docker build || exit`.

Reply:
https://linuxsimply.com/bash-scripting-tutorial/redirection-and-piping/piping/double-pipe

Reply:
Ooh, thanks for sharing! Didn't know about that one